docs-site/docs/configuration/ga4gh_wes.md

```diff
@@ -4,34 +4,74 @@ id: ga4gh_wes
 title: Connecting to a GA4GH WES server
 ---
 
-### Setup
+### Server Connection
 
-The following fields can be set in the rails credentials file.
+To connect IRIDA Next to a GA4GH Server, you will need to edit your rails credentials file. This file is encrypted so secret tokens are secure.
+
+Edit your production credentials file with the following command.
+
+`EDITOR="vim --nofork" bin/rails credentials:edit --environment production`
+
+[Read more about the rails credential file here](https://guides.rubyonrails.org/security.html#custom-credentials)
+
+The following fields can be set.
 
 ```yml
 ga4gh_wes:
+  server_url_endpoint: 'https://<some server url>/wes/1.01/'
   oauth_token: <some oauth token>
   headers: { '<some header key>': '<some header value>' }
-  server_url_endpoint: 'https://<some server url>/wes/1.01/'
 ```
 
-#### oauth_token
+Only `server_url_endpoint` is required by default, but depending on your GA4GH WES server configuration the others might also be needed.
+
+#### server_url_endpoint: *Required
+
+Set the server endpoint.
+
+When this is set, make sure to include the full endpoint. e.g. `https://subdomain.domain.tld/wes/1.01/`
+
+Example: If your endpoint for getting the Service Info path is `localhost:1122/service-info`, your endpoint is `localhost:1122`
+
+#### oauth_token: *Optional
 
 Bearer token for OAuth 2.0
 
-#### headers
+If your GA4GH WES server is using OAuth 2.0 for authentication, this field is required.
 
-Can be used to set new headers and override existing headers
+#### headers: *Optional
 
-This allows for additional authentication or server settings for your specific deployment of GA4GH WES
+Can be used to set new headers and override existing REST API headers.
 
-By default `{ 'Content-Type': 'application/json' }` is set.
+This allows for additional authentication or server settings for your specific deployment of GA4GH WES.
 
-#### server_url_endpoint
+By default `{ 'Content-Type': 'application/json' }` is set per the GA4GH WES standard.
```

Comment:
**nitpick (non-blocking):** There are actually no default headers, I removed that header when we updated to use multipart form encoding as per the GA4GH WES Spec.

Reply:
Fixed in https://github.com/phac-nml/irida-next/pull/628/commits/ccc745a870e8897af47c5cc4221ed50e20a12cd5